### PR TITLE
Merge from container-images/perl. Mostly update labels

### PR DESCRIPTION
**Pull request by phracek**
Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

This Pull Request updates LABELs in Dockerfile and Dockerfile.rhel7

It is merge from container-images/perl repo.

### CODE REVIEW

File: 5.20/Dockerfile [hunk 1]
```diff
@@ -19,14 +21,18 @@ and web programming."
...
+      io.openshift.s2i.scripts-url="image:///usr/libexec/s2i" \
+      io.s2i.scripts-url="image:///usr/libexec/s2i" \
+      name="centos/$NAME-$PERL_SHORT_VER-centos7" \
+      com.redhat.component="rh-$NAME$PERL_SHORT_VER-docker" \
+      version="$PERL_VERSION" \
       release="1" \
-      maintainer="SoftwareCollections.org <sclorg@redhat.com>"
+      maintainer="SoftwareCollections.org <sclorg@redhat.com>" \
+      help="For more information visit https://github.com/sclorg/s2i-$NAME-container" \
+      usage="s2i build <SOURCE-REPOSITORY> $NAME:5 <APP-NAME>"
```
Comment (omron93) [inline 5.20/Dockerfile]: No image with name `$NAME:5`. What about `centos/$NAME-$PERL_SHORT_VER-centos7:latest` instead?
(same in all Dockerfiles)

File: 5.20/Dockerfile.rhel7 [hunk 2]
```diff
@@ -31,7 +31,7 @@ LABEL summary="$SUMMARY" \
       release="1" \
       maintainer="SoftwareCollections.org <sclorg@redhat.com>" \
       help="For more information visit https://github.com/sclorg/s2i-$NAME-container" \
-      usage="s2i build <SOURCE-REPOSITORY> $NAME:5 <APP-NAME>"
+      usage="s2i build <SOURCE-REPOSITORY> centos/$NAME-$PERL_SHORT_VER-centos7:latest <APP-NAME>"
```
Comment (ppisar) [inline 5.20/Dockerfile.rhel7]: I don't think RHEL documentation (*.rhel7 dockerfiles) should refer to centos.

Comment (phracek) [reply 5.20/Dockerfile.rhel7]: Yeah, Copy-paste error.

File: 5.24/README.md [hunk 3]
```diff
@@ -9,6 +9,56 @@ The CentOS image is then available on [Docker Hub](https://hub.docker.com/r/cent
...
+
+* **`test/`**
+
+	This folder contains some sample applications you can use to test you image.
+
+    * **`sample-test-app/`**
+
+        A simple Perl application used for testing purposes by the [S2I](https://github.com/openshift/source-to-image) test framework.
+
+
```
[the remark this answers is not in the document]
Comment (phracek) [reply 5.24/README.md]: @ficap @ppisar I guess, repository organization should be add in all versions. Why it was removed? What rules do you mean?

[the remark this answers is not in the document]
Comment (pkubatrh) [reply 5.24/README.md]: @phracek The repository organisation section is not wanted inside version specific README files as these are what the user sees inside the images. It makes more sense to have this mentioned in the root README (even more so when the organisation is the same for all of the versioned folders).

File: 5.20/Dockerfile.rhel7 [hunk 4]
```diff
@@ -18,13 +20,18 @@ and web programming."
...
+      io.openshift.tags="builder,$NAME,$NAME$PERL_SHORT_VER" \
+      io.openshift.s2i.scripts-url="image:///usr/libexec/s2i" \
+      io.s2i.scripts-url="image:///usr/libexec/s2i" \
+      name="centos/$NAME-$PERL_SHORT_VER-centos7" \
+      com.redhat.component="rh-$NAME$PERL_SHORT_VER-docker" \
+      version="$PERL_VERSION" \
+      release="1" \
+      maintainer="SoftwareCollections.org <sclorg@redhat.com>" \
+      help="For more information visit https://github.com/sclorg/s2i-$NAME-container" \
+      usage="s2i build <SOURCE-REPOSITORY> centos/$NAME-$PERL_SHORT_VER-centos7:latest <APP-NAME>"
```
Comment (omron93) [inline 5.20/Dockerfile.rhel7]: `rhscl/$NAME-$PERL_SHORT_VER-rhel7`

File: 5.20/Dockerfile.rhel7 [hunk 5]
```diff
@@ -18,13 +20,18 @@ and web programming."
...
       io.openshift.expose-services="8080:http" \
-      io.openshift.tags="builder,perl,perl520" \
-      name="rhscl/perl-520-rhel7" \
-      com.redhat.component="rh-perl520-docker" \
-      version="5.20" \
-      release="1"
+      io.openshift.tags="builder,$NAME,$NAME$PERL_SHORT_VER" \
+      io.openshift.s2i.scripts-url="image:///usr/libexec/s2i" \
+      io.s2i.scripts-url="image:///usr/libexec/s2i" \
+      name="centos/$NAME-$PERL_SHORT_VER-centos7" \
```
Comment (omron93) [inline 5.20/Dockerfile.rhel7]: This is rhel based version.

**Comment by omron93** [comment]
[test]

**Comment by omron93** [comment]
@phracek Thanks for the fix.

And sorry for delayed reply, I forgot about this PR.

**Pull request merged**